prototypes/v2/src/controllers.ts

```diff
@@ -55,8 +56,7 @@ export module Controllers {
     } else {
       let qterm = removePunctuation(term);
 
-      let query =
-        "select rank, PRODUCT_ID from search where search match ? order by rank LIMIT ? OFFSET ?;";
+      let query = `select rank, PRODUCT_ID from search where search match ? order by ${sortBy} LIMIT ? OFFSET ?;`;
```

Comment:
Isn't this an injection issue?
```
?sortBy=rank%3BDROP%20TABLE%20products%3B--

Reply:
Yep, absolutely. I've shelved this as a WIP while I rethink it a bit.